spec/support/shared/examples/payload_cached_size_is_consistent.rb

```diff
@@ -157,7 +180,11 @@
         )
         expect(pinst.cached_size).to_not(be_nil)
         expect(pinst.dynamic_size?).to be(false)
-        expect(pinst.cached_size).to eq(pinst.generate_simple(opts).size)
+        if pinst.shortname =~ /6/
```

Comment:
It's cleaner code
```php
expect(pinst.cached_size).to eq(pinst.generate_simple(pinst.shortname =~ /6/ ? opts6 : opts).size)
```

Reply:
I respectfully disagree, ternary operators while providing less lines of code often make debugging an error harder.  While there is a time and place for usage of a ternary operator for me it is rarely to be consider "cleaner" code.